service/mcs/rest/src/main/java/eu/europeana/cloud/service/mcs/rest/DataSetRevisionsResource.java

```diff
@@ -62,37 +57,33 @@
      * @param representationName representation name.
      * @param revisionName       name of the revision
      * @param revisionProviderId provider of revision
-     * @param startCloudId       reference to next slice of result. If not provided,
+     * @param revisionTimestamp  timestamp used for identifying revision, must be in UTC format
+     * @param startFrom          reference to next slice of result. If not provided,
      *                           first slice of result will be returned.
-     * @return slice of representation version list.
+     * @return slice of cloud id with tags of the revision list.
      * @throws DataSetNotExistsException no such data set exists.
      * @summary get representation versions from a data set
      */
     @GET
     @Produces({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON})
-    @ReturnType("eu.europeana.cloud.common.response.ResultSlice<String>")
+    @ReturnType("eu.europeana.cloud.common.response.ResultSlice<CloudTagsResponse>")
     public Response getDataSetContents(@PathParam(P_PROVIDER) String providerId,
-                                       @PathParam(P_DATASET) String dataSetId,
-                                       @PathParam(P_REPRESENTATIONNAME) String representationName,
-                                       @PathParam(P_REVISION_NAME) String revisionName,
-                                       @PathParam(P_REVISION_PROVIDER_ID) String revisionProviderId,
-                                       @QueryParam(F_START_FROM) String startCloudId,
-                                       @QueryParam(F_LIMIT) @Min(1) @Max(10000) int limitParm)
```

Comment:
In fact it comes from the original implementation of query 7 so I did not touch it. However I removed the min and max annotations and when there is no limitParm given it comes with 0 value which is checked programmaticaly.